features/filesystem/bd/ExhaustibleBlockDevice.h

```diff
@@ -24,6 +24,10 @@
 
 #include "BlockDevice.h"
 
+enum {
+    BD_ERROR_ERASE_UNIT_WORN_OUT     = -3301,
```

Comment:
I'm confused. Why is this an anonymous enum that only has one entry?

Reply:
Followed the convention dictated by existing block devices, like [MBRBlockDevice](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/MBRBlockDevice.h#L31). My understanding is that these error codes are additions to the [ones defined in BlockDevice base class](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/BlockDevice.h#L27).

Reply:
👍 I figured as much, but wanted to be sure.

Reply:
I think the error should be removed. Real flash will not identify the error.

Reply:
Right. The error will be removed as it also breaks the LittleFS wear level test.